tools/README.md

```diff
@@ -0,0 +1,7 @@
+# Tools for Omnia
+
+## change_personality
+```
+change_personality k|s <node_list>
```

Comment:
probably should add long format also

Reply:
I will open a new issue for the enhancement. Are you referring to allowing FQDN on the arg list?